documentation/book/con-configuring-container-images.adoc

```diff
@@ -0,0 +1,20 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-mirror-maker.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='con-configuring-container-images-{context}']
+
+= Container images
+
+Use the `image` property to configure the container image used by the component.
+
+Overriding container images is recommended only in special situations where you need to use a different container registry.
```

Comment:
or when you want to use a customized image? maybe for debugging purposes a user could create a new Docker image FROM one of our image but adding a command line tool we don't provide out of box in order to do some debug ...

Reply:
Thanks. I've added that suggestion as another example. 

For example, if your network does not allow access to the container repository used by {ProductName}, you can copy the {ProductName} images or build them from the source.
However, if the configured image is not compatible with {ProductName} images, it might not work properly.

A copy of the container image might also be customized and used for debugging.